pkg/bundle/internal/ssa_client/patch.go

```diff
@@ -17,10 +17,65 @@ limitations under the License.
 package ssa_client
 
 import (
+	"encoding/json"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
+	coreapplyconfig "k8s.io/client-go/applyconfigurations/core/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	FieldManager = client.FieldOwner("trust-manager")
+)
+
+func GenerateConfigMapPatch(
+	configmapPatch *coreapplyconfig.ConfigMapApplyConfiguration,
+) (*corev1.ConfigMap, client.Patch, error) {
+	if configmapPatch == nil || configmapPatch.Name == nil || configmapPatch.Namespace == nil {
+		panic("configmapPatch must be non-nil and have a name and namespace")
+	}
+
+	// This object is used to deduce the name & namespace + unmarshall the return value in
+	configmap := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      *configmapPatch.Name,
+			Namespace: *configmapPatch.Namespace,
+		},
+	}
+
+	encodedPatch, err := json.Marshal(configmapPatch)
+	if err != nil {
+		return configmap, nil, err
+	}
+
+	return configmap, applyPatch{encodedPatch}, nil
+}
+
+func GenerateSecretPatch(
```

Comment:
@erikgb same here, can this be moved back to reduce diff?

Reply:
Done